raft/raft.go

```diff
@@ -101,6 +102,8 @@ var stmap = [...]string{
 	"StatePreCandidate",
 }
 
+var kronosNewRaft = os.Getenv("KRONOS_NEW_RAFT")
```

Comment:
Can you please use a name to mention this is a "leader election fix"?
Also, use something like: `var kronosNewRaft = os.Getenv("KRONOS_NEW_RAFT") == "true"`, so that you don't have to compare the value of this repeatedly.